src/test/resources/com/google/maps/PlaceDetailsQuay.json

```diff
@@ -1,5 +1,6 @@
 {
   "html_attributions": [],
+  
```

Comment:
Looks like there might be a missing changes here? Also, please clarify what the change is supposed to address.